Gruntfile.js

```diff
@@ -1221,39 +1177,51 @@ module.exports = function(grunt) {
   // Android
   // =========================================================================
 
-  registerTask(grunt, 'build_android', [
-    // Builds Android from scratch by recreating the Cordova environment.
+  // Recreates the CCA build environment.
+  registerTask(grunt, 'recreate_cca_env', [
     'exec:cleanAndroid',
     'build_cca',
-    'exec:ccaCreateDev',
-    'exec:ccaPlatformAndroidDev',
-    'exec:ccaAddPluginsAndroidDev',
-    'copy:cca_splash_dev',
-    'build_android_lite'
+    'exec:ccaCreate',
+    'exec:ccaPlatformAndroid',
+    'exec:ccaAddPluginsAndroid',
+    'copy:cca_splash',
   ]);
-  registerTask(grunt, 'build_android_lite', [
-    // Android build task that does not recreate the Cordova environment.
-    // Should only be used for building CCA modules and after running
-    // build_android, without cleaning, initially at least once.
+
+  // Can be used for building CCA modules and after running recreate_cca_env,
+  // without cleaning, initially at least once.
+  registerTask(grunt, 'android_debug_lite', [
     'build_cca',
-    'exec:ccaBuildAndroid',
+    'exec:ccaBuildAndroidDev',
     'exec:androidReplaceXwalkDev'
   ]);
-  registerTask(grunt, 'release_android', [
+
+  registerTask(grunt, 'android_debug', [
+    'recreate_cca_env',
+    'android_debug_lite'
+  ]);
+
+  // Can be used for building CCA modules and after running recreate_cca_env,
+  // without cleaning, initially at least once.
+  registerTask(grunt, 'android_release_lite', [
     'build_cca',
-    'copy:dist',
-    'exec:ccaCreateDist',
-    'exec:ccaPlatformAndroidDist',
-    'exec:ccaAddPluginsAndroidDist',
-    'copy:cca_splash_dist',
     'symlink:cca_keys',
-    'exec:ccaReleaseAndroid',
-    'exec:androidReplaceXwalkDist'
+    'exec:ccaBuildAndroidRelease',
+    'exec:androidReplaceXwalkRelease'
+  ]);
+
+  registerTask(grunt, 'android_release', [
+    'recreate_cca_env',
+    'android_release_lite'
+  ]);
+
+  registerTask(grunt, 'android', [
+    'android_debug',
+    'android_release'
```

Comment:
It seems to me that the second will overwrite the first because they are both outputting to the same directory. Is that not the case?